detox/local-cli/detox-create-e2e.js

```diff
@@ -0,0 +1,60 @@
+var fs = require('fs');
```

Comment:
add commander params to support multiple test runners (you can set mocha as default), and make it easy to add more templates for other test runners in the future.
This means moving all the templates to a different file (`templates.mocha.js` ?) and creating a switch like we have in `detox-test.js`